### ISSUE TITLE
How can I copy filament profiles to second printer (from P1P to P1S). Will calibrate them then again for new printer, but would be nice to copy the whole data first

### ISSUE THREAD

**Issue by KuroObi72** [SoftFever/OrcaSlicer#7327]
### Is there an existing issue for this feature request?

- [X] I have searched the existing issues

### Is your feature request related to a problem?

No problem, but would make thinks easier

### Which printers will be beneficial to this feature?

All

### Describe the solution you'd like

An import of cusomer filament profiles to a new printer. When I switch between the printers (P1P and new P1S) I would have to gve all the data in again. Of course for the new printer a calibration of the filament is necessary afterwards. 

### Describe alternatives you've considered

if not a new function, a why how to copy it on my computer would help

### Additional context

-

**Comment by thePunderWoman** [comment]
Yes, please add this! I have upgraded an existing printer, and this is considered a separate printer in Orca. I have tuned profiles that I'd like to just use on the new printer, but there's no way to copy them to the new one. Having to manually copy and paste all the values to the new printer is really tedious.
